### PR TITLE
LPD-20037 Allow users without permission to view the blog's images to still view the blog entry

### PR DESCRIPTION
**Pull request by jonathanmccann**
https://liferay.atlassian.net/browse/LPD-20037

If a user does not have permission to view a blog's cover image then an exception is thrown and the user can only see the title of the blog. Instead of having an exception bubble up these changes make it so that `null` is returned instead.

If the correct solution is that the user should not be able to see the blogs entry at all then please let me know and I can work on making those changes. However, to me, this makes the most sense as they have permission to view the entry just not the image.

### CODE REVIEW

**Comment by jonathanmccann** [comment]
ci:test:relevant

File: modules/apps/blogs/blogs-service/src/main/java/com/liferay/blogs/model/impl/BlogsEntryImpl.java [hunk 1]
```diff
@@ -126,6 +140,21 @@ public void setSmallImageType(String smallImageType) {
 		_smallImageType = smallImageType;
 	}
 
+	private FileEntry _getFileEntry(long fileEntryId) {
```
Comment (adolfopa) [inline modules/apps/blogs/blogs-service/src/main/java/com/liferay/blogs/model/impl/BlogsEntryImpl.java:143]: Can you rename this method to `_fetchFileEntry`? in general, `get*` methods should return a value or throw `NoSuch*` if there's none available.

**Review by adolfopa — CHANGES_REQUESTED** [review]

**Comment by marcogalluzzi** [comment]
@jonathanmccann please have a look to failing POSHI tests, they seem related.

**Comment by jonathanmccann** [comment]
ci:report:2548267254

**Comment by jonathanmccann** [comment]
ci:report:2548291796

**Comment by jonathanmccann** [comment]
ci:test:relevant

**Comment by marcogalluzzi** [comment]
ci:stop

**Comment by marcogalluzzi** [comment]
ci:test:stable

**Comment by marcogalluzzi** [comment]
ci:test:sf

**Comment by ambrinchaudhary** [comment]
ci:forward

**Pull request closed without merging**